### ISSUE TITLE
OpenMP codegen + nested loops = compiler error

### ISSUE THREAD

**Issue by manopapad** [StanfordLegion/legion#382]
When I compile the file in [openmp_nested_loop.rg.tar.gz](https://github.com/StanfordLegion/legion/files/1944582/openmp_nested_loop.rg.tar.gz) with `regent.py openmp_nested_loop.rg -fopenmp 1` I get:

```
src/terralib.lua:1123: Errors reported during typechecking.
/home/manolis/sw/legion/language/src/regent/codegen.t:7064: definition of variable with symbol $runtime is not in scope in this context
      var [domain] = c.legion_index_space_get_domain([cx.runtime], [is])
                                                       ^
/home/manolis/sw/legion/language/src/regent/codegen.t:7017: definition of variable with symbol $r is not in scope in this context
    is = `([value.value].impl.index_space)
                ^

stack traceback:
	[C]: in function 'error'
	src/terralib.lua:388: in function 'finishandabortiferrors'
	src/terralib.lua:3314: in function 'typecheck'
	src/terralib.lua:1123: in function 'defineobjects'
	/home/manolis/sw/legion/language/src/regent/codegen.t:7261: in function 'fn'
	src/asdl.lua:18: in function 'block'
	/home/manolis/sw/legion/language/src/regent/codegen.t:9082: in function 'thunk'
	/home/manolis/sw/legion/language/src/regent/std_base.t:1125: in function 'complete'
	/home/manolis/sw/legion/language/src/regent/std.t:3700: in function 'compile_tasks_in_parallel'
	/home/manolis/sw/legion/language/src/regent/std.t:3807: in function 'start'
	openmp_nested_loop.rg:20: in main chunk
```

**Comment by magnatelee** [comment]
Should be fixed with cc517fcb.

**Comment by manopapad** [comment]
Confirmed
